modules/javafx.web/src/main/native/Source/WTF/wtf/java/FileSystemJava.cpp

```diff
@@ -273,7 +273,7 @@ int readFromFile(PlatformFileHandle handle, void* data, int length)
     return result;
 }
 
-String pathGetFileName(const String& path)
+String pathFileName(const String& path)
```

Comment:
This method is used 2 times in file  modules/javafx.web/src/main/native/Source/WebCore/fileapi/FileCocoa.mm.
It needs correction.

Reply:
Indeed, now there are no more calls to pathGetFileName in the branch